monitoring/harmony-monitor/blockchain-watchdog/rpc.go

```diff
@@ -64,20 +82,41 @@ type SuperCommitteeReply struct {
 }
 
 type CommitteeInfo struct {
-	PolicyType    string            `json:"policy"`
-	ShardID       int               `json:"shard-id"`
-	MemberCount   int               `json:"count"`
-	Committee     []CommitteeMember `json:"committee-members"`
-	HarmonyPower  string            `json:"hmy-voting-power"`
-	StakedPower   string            `json:"staked-voting-power"`
-	TotalRawStake string            `json:"total-raw-staked"`
+	PolicyType      string            `json:"policy"`
+	MemberCount     int               `json:"count"`
+	Committee       []CommitteeMember `json:"committee-members"`
+	HarmonyPower    string            `json:"hmy-voting-power"`
+	StakedPower     string            `json:"staked-voting-power"`
+	TotalRawStake   string            `json:"total-raw-staked"`
 }
 
 type CommitteeMember struct {
-	Address        string `json:"earning-account"`
-	IsHarmonyNode  bool   `json:"is-harmony-slot"`
-	BLSKey         string `json:"bls-public-key"`
-	RawPercent     string `json:"voting-power-unnormalized,omitempty"`
-	VotingPower    string `json:"voting-power-%"`
-	EffectiveStake string `json:"effective-stake,omitempty"`
+	Address         string           `json:"earning-account"`
+	IsHarmonyNode   bool             `json:"is-harmony-slot"`
+	BLSKey          string           `json:"bls-public-key"`
+	RawPercent      string           `json:"voting-power-unnormalized,omitempty"`
+	VotingPower     string           `json:"voting-power-%"`
+	EffectiveStake  string           `json:"effective-stake,omitempty"`
+}
+
+// Last Cross Links RPC Reply
+type LastCrossLinkReply struct {
+	CrossLinks      []CrossLink      `json:"-"`
+}
+
+type CrossLink struct {
```

Comment:
Let's build watchdog statically against the main repo so we can get rid of these redundant data strucutres and just import them directly from main repo